File: csmpy/examples/example_without_yaml_file.py
from csmpy import ColorSchemeManager

# Define default colors using color names
DEFAULT_COLORS = {
    'logo': 'red',
    'text': 'blue',
    'result': 'light_white',
}

# Load all colors
all_colors = ColorSchemeManager.load_colors(DEFAULT_COLORS)

# Example usage
logo_color = all_colors['logo']
text_color = all_colors['text']
result_color = all_colors['result']

print(f"This is a {logo_color} example of {text_color} using {result_color} colors.")

File: csmrb/examples/example_without_yaml_file.rb
require './lib/csm'

# Set the default color scheme
DEFAULT_COLORS = {
  'logo' => 'red',
  'text' => 'blue',
  'result' => 'light_white'
}.freeze

all_colors = ColorSchemeManager::load_colors(DEFAULT_COLORS)
puts "All colors: #{all_colors}"
puts "This is a #{'logo'.send(all_colors['logo'])} example of #{'text'.send(all_colors['text'])} using #{'result'.send(all_colors['result'])} colors."
